acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -7,69 +7,20 @@
-        if ENV['SERVER_VERSION'].to_i < 5
-          logger.info "EC2 master found: Installing nightly build of puppet-agent repo to satisfy puppetserver dependency."
-          on(master, "rpm -Uvh https://yum.puppetlabs.com/puppetlabs-release-pc1-el-#{version}.noarch.rpm")
-        else
-          logger.info "EC2 master found: Installing nightly build of puppet-agent repo to satisfy puppetserver dependency."
-          on(master, "rpm -Uvh https://yum.puppetlabs.com/puppet5-release-el-#{version}.noarch.rpm")
-        end
-      else
-        logger.info "EC2 master found: Installing nightly build of puppet-agent repo to satisfy puppetserver dependency."
-        install_repos_on(master, 'puppet-agent', 'nightly', 'repo-configs')
-        install_repos_on(master, 'puppetserver', 'nightly', 'repo-configs')
-      end
-
-      master.install_package('puppetserver')
-
-      logger.info "EC2 master found: Installing #{ENV['SHA']} build of puppet-agent."
-      # Upgrade installed puppet-agent with targeted SHA.
-      opts = {
-        :puppet_collection => 'PC1',
-        :puppet_agent_sha => ENV['SHA'],
-        :puppet_agent_version => ENV['SUITE_VERSION'] || ENV['SHA'] ,
-        :dev_builds_url => "http://builds.delivery.puppetlabs.net"
-      }
-
-      copy_dir_local = File.join('tmp', 'repo_configs', master['platform'])
-      release_path_end, release_file = master.puppet_agent_dev_package_info( opts[:puppet_collection], opts[:puppet_agent_version], opts)
-      release_path = "#{opts[:dev_builds_url]}/puppet-agent/#{opts[:puppet_agent_sha]}/repos/"
-      release_path << release_path_end
-      fetch_http_file(release_path, release_file, copy_dir_local)
-      scp_to master, File.join(copy_dir_local, release_file), master.external_copy_base
-      on master, "rpm -Uvh #{File.join(master.external_copy_base, release_file)} --oldpackage --force"
-    else
-      fail_test("EC2 master found, but it was not an `el` host: The specified `puppet-agent` build (#{ENV['SHA']}) cannot be installed.")
-    end
+  if ENV['SERVER_VERSION'].nil? || ENV['SERVER_VERSION'] == 'latest'
+    install_puppetlabs_dev_repo(master, 'puppetserver', 'latest', nil, :dev_builds_url => "http://nightlies.puppet.com")
+    master.install_package('puppetserver')
   else
-    if ENV['SERVER_VERSION'].nil? || ENV['SERVER_VERSION'] == 'latest'
-      server_version = 'latest'
-      server_download_url = "http://nightlies.puppet.com"
-      install_puppetlabs_dev_repo(master, 'puppetserver', server_version, nil, :dev_builds_url => server_download_url)
-      master.install_package('puppetserver')
-    else
-      server_version = ENV['SERVER_VERSION']
-      dev_builds_url = ENV['DEV_BUILDS_URL'] || 'http://builds.delivery.puppetlabs.net'
-      install_from_build_data_url('puppetserver', "#{dev_builds_url}/puppetserver/#{server_version}/artifacts/#{server_version}.yaml", master)
-    end
+    server_version = ENV['SERVER_VERSION']
```

Comment:
This may break acceptance when it gets merged up, but I'm not sure. I think for most of the pipelines `SERVER_VERSION` is set to `latest`